about.html

```diff
@@ -34,13 +37,13 @@
             <li><a href="#">Join</a></li>
             <li><a href="#">Sponsor</a></li>
             <li><a href="#">News</a></li>
-            <li id="compaign"><a href="#">Compaign</a></li>
         </ul>
+        <p id=compaign>AIMCompaign</p>
 
     </div>
 
     <div class="header-info-about">
-        <img src="./images/header-bg-about-page.jpg">
+        <img src="./images/header-bg-about-page.jpg" alt="medicebackground">
         <h1>MEDICINE FORUM IN AZERBAIJAN </h1>
```

Comment:
- [ ] **The title text in your about section is hidden behind the block of text. It also disappears on the larger screens. Please fix this.**
![bouggd](https://user-images.githubusercontent.com/59168430/125165603-44879b00-e1b5-11eb-842d-7c7e02b5305f.gif)

Reply:
Done -commit "About page header section size fizing"